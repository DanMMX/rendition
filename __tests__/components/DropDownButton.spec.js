/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import Provider from '../../src/components/Provider'
import DropDownButton from '../../src/components/DropDownButton'

test('DropDownButton renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <DropDownButton primary label={<div>DropDown</div>}>
        <div>Item</div>
        <div>Item</div>
        <div>Item</div>
        <div>Item</div>
      </DropDownButton>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
