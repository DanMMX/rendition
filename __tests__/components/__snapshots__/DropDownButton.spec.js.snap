// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDownButton renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-eNQAEJ fTusFU"
  >
    <span>
      <button
        className="sc-ckVGcZ emIugb sc-gqjmRU bCjhLA sc-htoDjs hfajhw"
      >
        <div>
          DropDown
        </div>
      </button>
      <button
        className="sc-dxgOiQ sZxT sc-gqjmRU bCjhLA sc-htoDjs hfajhw"
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height="1em"
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "color": undefined,
              "verticalAlign": "middle",
            }
          }
          viewBox="0 0 40 40"
          width="1em"
        >
          <g>
            <path
              d="m31.4 15.7q0 0.6-0.5 1l-10 10q-0.4 0.4-1 0.4t-1-0.4l-10-10q-0.4-0.4-0.4-1t0.4-1 1-0.4h20q0.6 0 1 0.4t0.5 1z"
            />
          </g>
        </svg>
      </button>
    </span>
  </div>
</div>
`;
